src/styles/blocks/card.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  width: $width-card;
+  padding: 32px 16px 16px;
+
+  border: 1px solid $border-card-colors;
+  border-radius: 5px;
+
+  &__logo {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
```

Comment:
This selector has different styles, font-size and something else (maybe)
<img width="179" alt="image" src="https://user-images.githubusercontent.com/32794096/200826294-34e05277-08a1-4d0e-9511-5c93eb1ef9fc.png">